packages/patternfly-4/src/components/layout.js

```diff
@@ -126,7 +118,6 @@ class Layout extends React.Component {
       return (
         <>
           <Header siteTitle={data.site.siteMetadata.title} />
-          <BackgroundImage src={bgImages} />
```

Comment:
Are we getting rid of the background for all pages?

Reply:
There isn't a transparent navigation anymore and all of the pages have either a grey or white background. Figured it was safe to remove, since it just adds to page load times without being used.

Reply:
Yes I agree, good to remove 👍